docs/content/integrations/dbt.mdx

```diff
@@ -1,380 +1,159 @@
-    else:
-        context.resources.dbt.run(models=["tag:other"])
-```
-
-#### Using a different dbt profile for different dagster modes
-
-Dagster supports creating [multiple jobs from the same graph](/concepts/ops-jobs-graphs/jobs#from-a-graph). dbt has a similar concept, [profiles](https://docs.getdbt.com/dbt-cli/configure-your-profile). You might want to run a dev version of your graph that targets the development-specific dbt profile, and then have a prod version that runs using the prod dbt profile. This example shows how to accomplish this.
-
-```python file=/integrations/dbt.py startafter=start_marker_dbt_cli_profile_modes endbefore=end_marker_dbt_cli_profile_modes dedent=4
-from dagster_dbt import dbt_cli_resource, dbt_run_op
+You can configure this resource and add it to your dbt assets:
 
-from dagster import graph
-
-@graph
-def my_dbt():
-    dbt_run_op()
-
-my_dbt_graph_dev = my_dbt.to_job(
-    resource_defs={
-        "dbt": dbt_cli_resource.configured(
-            {"project_dir": "path/to/dbt/project", "profile": "dev"}
-        )
-    }
-)
+```python
+from dagster import with_resources
+from dagster_dbt import dbt_cli_resource
 
-my_dbt_graph_prod = my_dbt.to_job(
-    resource_defs={
+dbt_assets = with_resources(
+    load_assets_from_dbt_project(...),
+    {
         "dbt": dbt_cli_resource.configured(
-            {"project_dir": "path/to/dbt/project", "profile": "prod"}
+            {"project_dir": "path/to/dbt_project"}
         )
     }
 )
 ```
 
-#### Invoking multiple dbt commands in the same job
-
-Sometimes, you'll want to run multiple different dbt commands in the same job. The `dbt_cli_resource` makes this convenient, as you only need to configure your dbt resource once, and all of that configuration will already be set for any ops that are using this resource.
-
-One common use case would be to first run `dbt run` to update all of your models, and then run `dbt test` to check that they all are working as expected, seen below.
-
-```python file=/integrations/dbt.py startafter=start_marker_dbt_cli_run_after_another_op endbefore=end_marker_dbt_cli_run_after_another_op dedent=4
-from dagster_dbt import dbt_cli_resource, dbt_run_op, dbt_test_op
-
-from dagster import job
-
-my_dbt_resource = dbt_cli_resource.configured(
-    {"project_dir": "path/to/dbt/project"}
-)
-
-@job(resource_defs={"dbt": my_dbt_resource})
-def my_dbt_job():
-    dbt_test_op(start_after=dbt_run_op())
-```
-
-## Using a dbt RPC server in a Dagster job
+## Scheduling dbt jobs
 
-This integration provides two separate resources to help run commands against a [dbt RPC server](https://docs.getdbt.com/reference/commands/rpc). The `dbt_rpc_resource` will fire off requests against the server and return immediately, while the `dbt_rpc_sync_resource` will poll the server until each request completes.
+Once you have your dbt assets, it's simple to define a job that runs some or all of these assets on a schedule
```

Comment:
I'm sorta against including words like 'simple' or 'easy' in documentation because for some folks, it *isn't* easy and I don't want them to be frustrated. It also tends to make copy wordier than it needs to be.

What about something like:

> Once your dbt assets are ready, you can define a job that runs some or all of the assets on a schedule: